CHANGELOG.md

```diff
@@ -12,8 +12,35 @@ Mapbox welcomes participation and contributions from everyone.
 
 # 10.3.0 February 7, 2022
 
+## Features ✨ and improvements 🏁
+* Improve performance for symbol layout rendering in continuous mode. ([#1105](https://github.com/mapbox/mapbox-maps-android/pull/1105))
+* Introduce metadata setter API for the legacy offline region. ([#1105](https://github.com/mapbox/mapbox-maps-android/pull/1105))
```

Comment:
this was introduced in https://github.com/mapbox/mapbox-maps-android/releases/tag/android-v10.3.0-rc.1, do we want to state it again in the v10.3.0 changelog?

Reply:
@pengdev AFAIR we state delta of changes between last _stable_ release for the stable release. So we should have all the changes here since v10.2.0 letting majority users easily migrate between stable versions.